server/Mistakes.Journal.Api/Api/Mistakes/Controllers/WeatherController.cs

```diff
@@ -0,0 +1,89 @@
+﻿
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Mistakes.Journal.Api.Api.Mistakes.Mappers;
+using Mistakes.Journal.Api.Api.Mistakes.WebModels;
+using Mistakes.Journal.Api.Api.Shared;
+using Newtonsoft.Json.Linq;
+
+namespace Mistakes.Journal.Api.Api.Mistakes.Controllers
+{
+    [ApiController]
+    [Route("/api/weather")]
+    public class WeatherController : ControllerBase
+    {
+        private const string pwRequestPattern = "data/2.5/weather?lat={0}&lon={1}&appid={2}&units=metric";
+        private const string ssRequestPattern = "json?lat={0}&lng={1}&formatted=0";
+
+        private readonly Uri owmApiUri = new Uri("http://api.openweathermap.org");
+        private readonly Uri ssApiUri = new Uri("https://api.sunrise-sunset.org");
+
+        private readonly string owKey = Environment.GetEnvironmentVariable("OPEN_WEATHER_API_KEY");
```

Comment:
Private variables should be prefixed with `_`.

Reply:
ok, we can adopt this convention